src/components/Postblock.vue

```diff
@@ -0,0 +1,126 @@
+<script setup>
+</script>
+
+<template>
+    <div class="top_image_post_block">
```

Comment:
classes do not follow BEM convention - it should be  all starting with postblock as Block, than element 
postblock__image, postblock__info, postblock__info-header etc